src/Nonlinear/ReverseAD/Coloring/topological_sort.jl

```diff
@@ -0,0 +1,106 @@
+# Modified from Graphs.jl.
```

Comment:
Was it modified from the original Graphs.jl or from the current one? It can be confusing given that LightGraphs got renamed as Graphs

Reply:
Does the `Copyright (c) 2012` give it away?

Reply:
yes it works I guess